core/http/routes/ui_gallery.go

```diff
@@ -189,6 +189,62 @@ func registerGalleryRoutes(app *fiber.App, cl *config.ModelConfigLoader, appConf
 		return c.SendString(elements.StartModelProgressBar(uid, "0", "Installation"))
 	})
 
+	app.Post("/browse/config/model/:id", func(c *fiber.Ctx) error {
+		galleryID := strings.Clone(c.Params("id")) // note: strings.Clone is required for multiple requests!
+		log.Debug().Msgf("UI job submitted to get config for : %+v\n", galleryID)
+
+		models, err := gallery.AvailableGalleryModels(appConfig.Galleries, appConfig.SystemState)
+		if err != nil {
+			return err
+		}
+
+		model := gallery.FindGalleryElement(models, galleryID)
+		if model == nil {
+			return fmt.Errorf("model not found")
+		}
+
+		config, err := gallery.GetGalleryConfigFromURL[gallery.ModelConfig](model.URL, appConfig.SystemState.Model.ModelsPath)
+		if err != nil {
+			return err
+		}
+
+		// Save the config file
+		_, err = gallery.InstallModel(appConfig.SystemState, model.Name, &config, model.Overrides, nil, false)
```

Comment:
I'm lost on what this endpoint is supposed to do - why saving the config file again? we had it already in memory, and was loaded. what am I missing here?

Reply:
This saves the config file *without* redownloading the model weights.  E.g., if I already have the model, but I just want the config file to make it work.